app/assets/stylesheets/active_admin/print.scss

```diff
@@ -114,48 +105,24 @@ table {
   text-align: left;
   width: 100%;
 
-  thead {
-    display: table-header-group;
-
-    th {
-      background: none;
-      border-bottom: medium solid black;
-      font-weight: bold;
-
-      a{
-        text-decoration: none;
-      }
-    }
+  tr {
+    page-break-inside: avoid;
   }
 
-  th, td {
-    padding: .5em 1em;
-
-    .member_link {
-      display: none;
-    }
+  th {
+    font-weight: bold;
   }
 
+  th,
   td {
-    border-bottom: thin solid black;
-  }
-
-  tr{
-    page-break-inside: avoid;
+    border-bottom: 1px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.